servicemesh-install/topics/install.adoc

```diff
@@ -10,12 +10,18 @@ The {ProductShortName} installation process introduces a Kubernetes operator to
 == Installing the operator
 The following steps will install the {ProductShortName} operator into an existing {product-title} 3.10 installation; they can be executed from any host with access to the cluster.  Please ensure you are logged in as a cluster admin before executing the following commands.
 
-You can find the https://github.com/Maistra/openshift-ansible/tree/maistra-0.1.0-ocp-3.1.0-istio-1.0.0/istio[operator templates on GitHub].
+You can find the https://github.com/Maistra/openshift-ansible/tree/maistra-0.2.0-ocp-3.1.0-istio-1.0.2/istio[operator templates on GitHub].
```

Comment:
@vikram-redhat, are you ok with this GitHub reference?

Reply:
Hi Kevin - @knrc - is this the upstream for ServiceMesh?

Reply:
@vikram-redhat Yes it is.

Reply:
@knrc - is there a RH supported link instead? If not, is it confirmed within these docs that Service Mesh is not supported by RH and that this is dev preview only? If yes, we can approve this link.

Reply:
@vikram-redhat There is no product/supported link as of yet and this release is described as a tech preview within the docs and the images.

Reply:
Thanks Kevin (@knrc)! 

@kalexand-rh - yes we can accept this link.

Reply:
Thanks Vikram! When Kevin clears the merge errors, I'll merge this PR. :)